.github/CODEOWNERS

```
@@ -126,6 +127,7 @@
 /pkg/compliance/                        @DataDog/container-integrations
 /pkg/kubestatemetrics                   @DataDog/container-integrations
 /pkg/security/                          @DataDog/agent-security
+/pkg/snmp                               @DataDog/agent-integrations
```

```suggestion
/pkg/snmp/                               @DataDog/agent-integrations
```
Otherwise this only matches a file named `snmp` in the `/pkg/` folder.

---

@KSerrania Thx for the suggestion.

---

cc @DataDog/container-integrations about `/pkg/kubestatemetrics` line that does not end with `/`